Installer/output-build-json.ps1

```diff
@@ -29,7 +30,7 @@ $jsonTemplate = @"
   "assets": [
     {
       "name": "Ziti.Desktop.Edge.Client-${version}.exe",
-      "browser_download_url": "${url}/${version}/Ziti.Desktop.Edge.Client-${version}.exe"
+      "browser_download_url": "${url}/${version}${versionQualifier}/Ziti.Desktop.Edge.Client-${version}.exe"

```

Comment:
versionQualifier let's me add the "-alpha" qualifer to the url, just saves me a bit of time making these releases